next.js/src/app/Sidebar/sidebar.tsx

```diff
@@ -0,0 +1,49 @@
+'use client'
+
+import account from "./icons/person.svg";
+import logout from "./icons/box-arrow-left.svg";
+import calender from "./icons/calendar-date.svg";
+import settings from "./icons/gear.svg";
+import clock from "./icons/clock.svg";
+import view from "./icons/view-stacked.svg";
+import SideBarTab from "./SideBarTab";
+import { useState } from "react";
+import Image from "next/image";
+import list from "./icons/list.svg"
+
+export default function SideBar() {
+  let name = "Bert Bertson";
+  const [statusValue, setStatusValue] = useState(true)
```

Comment:
```suggestion
  const [statusValue, setStatusValue] = useState<boolean>(true)
```
Aangezien jullie typescript gebruiken kun je het beste bij dit soort definities expliciet de type doorgeven. Vaak kan typescript dat zelf wel ontcijferen, alleen het is voor andere mensen leesbaarder als je het expliciet definieert